### DISCUSSION TITLE
Reverse-Engineering YouTube

### DISCUSSION THREAD

**Discussion by giscus** [Tyrrrz/Tyrrrzme#30] [Blog]
# Reverse-Engineering YouTube

Oleksii Holub (@tyrrrz) is a software developer, open-source maintainer, tech blogger and conference speaker

https://tyrrrz.me/blog/reverse-engineering-youtube

**Comment by short443** [comment]
Hello, do you or anyone know if it is possible to set a start time for the video/audio file directly in the url like "&start=" does in normal YouTube link? Thus starting playback from a predetermined time of the video.

**Comment by alexanderavezbakiyev-png** [comment]
Youtube url isn't turn off profanity in content . Need turn off profanity in url ,some video is turn off ,need all turn off profanity signal all video kindly your assistance.  Fix youtube morphe issue of background play screen turn off need fix it please
